Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/Http.rst

```diff
@@ -205,52 +205,25 @@ still handled even if the new component cancels the current chain.
-respective comments for getting an idea about the available functions. That being said, we'll pick a few important
-methods which may need some further explanation.
-
-Constructing a Request
-~~~~~~~~~~~~~~~~~~~~~~
-
-You can, in theory, create a new ``Request`` instance by simply using the ``new`` operator and passing the required
-arguments to the constructor. However, there are two static factory methods which make life much easier. We recommend
-using these instead of the low-level constructor method.
-
-.. warning::
-
-	You should only create a ``Request`` manually if you want to send out requests or if you know exactly what you are
-	doing. The created ``Request`` will not have any ``HTTP Components`` affect him and might therefore lead to
-	unexpected results, like the trusted proxy headers ``X-Forwarded-*`` not being applied and the ``Request`` providing
-	wrong protocol, host or client IP address.
-	If you need access to the **current** HTTP ``Request`` or ``Response``, instead inject the ``Bootstrap`` and
-	get the ``HttpRequest`` and ``HttpResponse`` through the ``getActiveRequestHandler()``.
-  Note though that those two objects are immutable, so if you need to change them, you need to in turn tell the current
-  ``ComponentContext`` that they need to be replaced through the ``->replaceHttpRequest(..)`` and ``->replaceHttpResponse(..)``
-  methods respectively. You can get the current ``ComponentContext`` from the active ``RequestHandler``.
-
-create()
-~~~~~~~~
-
-The method ``create()`` accepts an URI, the request method, arguments and a few more parameters and returns a new
-``Request`` instance with sensible default properties set. This method is best used if you need to create a new
-``Request`` object from scratch without taking any real HTTP request into account.
-
-createFromEnvironment()
-~~~~~~~~~~~~~~~~~~~~~~~
-
-The second method, ``createFromEnvironment()``, take the environment provided by PHP's superglobals and specialized
-functions into account. It creates a ``Request`` instance which reflects the current HTTP request received from the web
-server. This method is best used if you need a ``Request`` object with all properties set according to the current
-server environment and incoming HTTP request.
-Note though, that you should not expect this ``Request`` to match the current ``Request``, since the latter will still
-have been affected by some ``HTTP Components``. If you need the **current** Request, get it from the ``RequestHandler`` instead.
+In the PSR-7 specification, a distinction is made between two different types of requests - incoming (``ServerRequest``)
+and outgoing (``Request``). Whenever you want to make an outgoing request, you can easily use for example Guzzle's
+``Request`` class constructor with the respective arguments for method, uri, etc. and then pass that to e.g. a PSR-18
+Http Client implementation.
```

Comment:
We need to make our own `Browser` PSR-18 compatible at some point. Right now it still uses a `ServerRequest` though in order to provide easy request forwarding in functional tests.